packages/store/src/workspace/page.ts

```diff
@@ -258,6 +258,9 @@ export class Page extends Space<PageData> {
     parent?: BaseBlockModel | string | null,
     parentIndex?: number
   ): string {
+    if (this.awareness.getFlag('readonly')) {
```

Comment:
**Caution**: when you only enable read-only in the store but not on the block page. Press 'Enter' in the paragraph could cause unexpected result

Reply:
which means you need set read-only value both in page and store